src/__experimental__/components/label/label.style.js

```diff
@@ -8,6 +8,17 @@ const LabelStyle = styled.label`
   display: block;
   font-weight: 600;
 
+  ${({
+    isRequired
+  }) => isRequired && css`
+    ::after {
+      content: '*';
+      color: #C7384F;
```

Comment:
Please define the colour in the `theme` config.

Reply:
Done